homeassistant/components/homeassistant/translations/en.json

```
@@ -1,3 +1,17 @@
 {
-    "title": "Home Assistant"
+    "system_health": {
+        "info": {
+            "arch": "CPU Architecture",
+            "dev": "Development",
+            "docker": "Docker",
+            "hassio": "HassOS",
```

```suggestion
            "hassio": "Has Supervisor",
```

This is True if `hassio` is in  `hass.config.components`

---

Probably just set to `Supervisor` then. 
```suggestion
            "hassio": "Supervisor",
```